src/metabase/models/serialization.clj

```diff
@@ -748,10 +752,13 @@
     (let [model-name (name model)
           model      (t2.model/resolve-model (symbol model-name))
           entity     (t2/select-one model (first (t2/primary-keys model)) id)
-          path       (mapv :id (generate-path model-name entity))]
-      (if (= (count path) 1)
-        (first path)
-        path))))
+          path       (when entity (mapv :id (generate-path model-name entity)))]
```

Comment:
This line would look quite nice with `some->>` I think 😄

Reply:
to be honest I really want `when entity` to stand out here, logic-wise...

Reply:
I just don't like reading from left to right for a bit, then right to left. Maybe put a line break then? 😋